lib/mix/lib/releases/assembler.ex

```diff
@@ -603,6 +603,7 @@ defmodule Mix.Releases.Assembler do
 
   defp generate_vm_args(%Release{profile: %Profile{vm_args: path}} = rel) do
     Shell.debug("Generating vm.args from #{Path.relative_to_cwd(path)}")
+    path = Path.expand(path)
```

Comment:
We should probably move this to the config handler, since we'll apply the same expansion to all paths where we'll be using as a source for a copy or link.

Reply:
I think actually I'll do that in a different commit, this is good for now